ArdentHQ/arkvault
# deps(react): fix failing in transaction domain

## Summary






The PR fixes the failing tests & coverage in the transaction domain after the react update to 18.

## QA Checklist
The following checklist is produced by the failing assertions in tests that needed refactoring. The application code is not changed, only the tests, but because react update to 18 is an integral change, it would be nice to double-check the following:

-   [ ]  Add a recipient with an invalid address (in send-transfer multiple form). It should validate the recipient address correctly, the button should stay disabled for invalid addresses.
-   [ ]  Send a transfer with a Ledger wallet and see the ledger loading screens.
-   [ ]  Change fees to the transaction and see them applied in the next steps. Enter very low fees and see the error/warning.
-   [ ]  Submit a multi-signature transaction, using the `enter` keyword instead of the button in the modal.
-   [ ]  Add multi-signature participants and check if the address validation works.
-   [ ]  Check if the musig signature statutes are displayed properly in musig detail modal.
-   [ ]  Export wallet transactions (default settings), and make sure the loading state is displayed.
-   [ ]  Create a musig registration and make sure the success screen is showing the correct values (addresses, participants etc).
-   [ ]  Send Delegate Resignation transaction
-   [ ]  Send Ipfs transaction
-   [ ] Send MuSig Registration with a multi-signature with a ledger wallet.
-   [ ] Send a single transfer from the navigation bar, and make sure the network selection works in the first step.
-   [ ] Send a vote transaction
-   [ ] Send a vote & transaction (combined)



## Checklist
